.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Build and push Stun server Docker image on release
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    if: ${{ github.repository == 'home-assistant/stun' }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4.1.7
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3.2.0
```

Comment:
Is this really needed for an amd64 only build?

Reply:
No, not right now. We can add it back If we decide to use Arm at some point.